web/templates/stream_settings/stream_members.hbs

```diff
@@ -20,4 +20,5 @@
         {{> stream_members_table .}}
     </div>
 </div>
+<div class="subscriber_list_settings_loading"></div>
```

Comment:
Let's not add new classes with snake case when we can avoid it.

Reply:
Sure, fixed!